spec/controllers/v2/concerns/collection_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# The `parents` parameter is required when testing nested controllers, and it should contain
+# an ordered list of reflection symbols similarly to how they are defined in the routes. It is
+# also required to set the `extra_params` variable in a let block and pass all the parent IDs
+# there as a hash. For example:
+# ```
+# it_behaves_like 'collection', :security_guide, :profile do
+#   let(:extra_params) { { security_guide_id: 123, profile_id: 456 } }
+# end
+# ```
+#
+# In a non-nested case this example can be simply included as:
+# ```
+# it_behaves_like 'collection'
+# ```
+#
+RSpec.shared_examples 'collection' do |*parents|
```

Comment:
Tha `parents` is no longer needed here and please adjust the documentation above.